website/frontend/config/urls.js

```diff
@@ -1,13 +1,9 @@
 import { stripTrailingSlash } from './utils';
 
-const BASE_AIRQLOUDS_URL = stripTrailingSlash(
-  process.env.REACT_APP_BASE_AIRQLOUDS_URL || process.env.REACT_NETMANAGER_BASE_URL,
-);
+const BASE_AIRQLOUDS_URL = stripTrailingSlash("https://staging-platform.airqo.net/api/v1/");
 
 export const AIRQLOUD_SUMMARY = `${BASE_AIRQLOUDS_URL}/devices/airqlouds/summary?tenant=airqo`;
 
-const BASE_NEWSLETTER_URL = stripTrailingSlash(
-  process.env.REACT_APP_BASE_NEWSLETTER_URL || process.env.REACT_NETMANAGER_BASE_URL,
-);
+const BASE_NEWSLETTER_URL = stripTrailingSlash("https://staging-platform.airqo.net/api/v1/");
```

Comment:
Newsletter url routes to the platform?

Reply:
@ja-odur should I leave these changes in the urls file or I should change it back?

Reply:
@Codebmk Don't check this in. This was a quick windows hack. We will get a better solution

Reply:
@ja-odur I have removed them. Thanks.